src/stream/src/common/table/state_table.rs

```diff
@@ -52,6 +52,8 @@ use tracing::trace;
 
 use crate::executor::{StreamExecutorError, StreamExecutorResult};
 
+const STATE_CLEANING_PERIOD_EPOCH: usize = 5;
```

Comment:
Can you add some comments to justify this magic number? Why not 10 or 100?

@Little-Wallace what do you think the magic number should be considering the performance?

We don't have to do every epoch and the cost is some operation to storage. 

Perhaps we are aiming that one round of state cleaning should be able to complete before next one is initiated?

Reply:
> Can you add some comments to justify this magic number? Why not 10 or 100?
> 
> @Little-Wallace what do you think the magic number should be considering the performance?
> 
> We don't have to do every epoch and the cost is some operation to storage.
> 
> Perhaps we are aiming that one round of state cleaning should be able to complete before next one is initiated?

Sorry but I have no idea.
perhaps we should set a time interval and use time interval / barriers frequency here.

Reply:
Sounds, good. Let's add some TODO comments that this number arbitrary and we may want to improve this choice in the future.

Reply:
> arbitrary and we may want to improve this choice in the future.

fixed